src/java/pom.ts

```diff
@@ -345,6 +402,16 @@ export class Pom extends Component {
       repositories: { repository: this.repositories },
     };
   }
+
+  private synthPluginRepositories() {
+    if (this.pluginRepositories.length === 0) {
+      return;
+    }
+
+    return {
+      pluginRepositories: { pluginRepositories: this.pluginRepositories },
```

Comment:
I think it should be this?
```suggestion
      pluginRepositories: { pluginRepository: this.pluginRepositories },
```